requirements-base-runtime.txt

```diff
@@ -113,7 +108,6 @@ markupsafe==2.1.5
     #   werkzeug
 mdurl==0.1.2
     # via markdown-it-py
-moto-ext==5.0.5.post1
```

Comment:
@bentsku as discussed this PR removes the moto-ext dependency in the base-runtime which was added by mistake with #10638.